decidim-core/app/cells/decidim/tos_page/refuse_btn_modal.erb

```diff
@@ -2,22 +2,39 @@
   <%= t("refuse.modal_button", scope: "decidim.pages.terms_and_conditions") %>
 </button>
 
-<div id="tos-refuse-modal" class="reveal" data-reveal aria-labelledby="#{modal_title}" aria-hidden="true" role="dialog">
-  <h2>
-    <%= t("refuse.modal_title", scope: "decidim.pages.terms_and_conditions") %>
-  </h2>
+<div id="tos-refuse-modal" class="reveal" data-reveal aria-labelledby="<%= t("refuse.modal_title", scope: "decidim.pages.terms_and_conditions") %>" aria-hidden="true" role="dialog">
+  <div class="reveal__header">
+    <h3 class="reveal__title">
+      <%= t("refuse.modal_title", scope: "decidim.pages.terms_and_conditions") %>
+    </h3>
 
-  <p>
-    <%= t("refuse.modal_body", scope: "decidim.pages.terms_and_conditions", data_portability_path: decidim.data_portability_path, delete_path: decidim.delete_account_path) %>
-  </p>
+    <button class="close-button" data-close aria-label="<%= t("refuse.modal_close", scope: "decidim.pages.terms_and_conditions") %>"
+      type="button">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+
+  <div class="row">
+    <div>
+      <p>
+        <%= t("refuse.modal_body", scope: "decidim.pages.terms_and_conditions", data_portability_path: decidim.data_portability_path, delete_path: decidim.delete_account_path) %>
+      </p>
+    </div>
 
-  <div class="row column flex-center">
-    <%= button_to decidim.destroy_user_session_path, method: :delete, class: "clear button secondary button--nomargin small" do %>
-      <%= t("refuse.modal_btn_exit", scope: "decidim.pages.terms_and_conditions") %>
-    <% end %>
+    <div class="columns medium-10 medium-centered">
```

Comment:
Remove this `div` for refuse modal. Since we have two buttons, if you apply `medium-10`, you're shrinking the row.
I'd say, in that case, you must leave the markup as it was.

Reply:
If you're reusing code for both modal contents, then, do not apply that `div.columns.medium-10.medium-centered` for the modal buttons (it seems we have always 2 buttons)